internal/github/team.go

```diff
@@ -0,0 +1,18 @@
+package github
+
+import (
+	"context"
+
+	gh "github.com/google/go-github/v33/github"
+)
+
+func TeamExist(cli *gh.Client, ctx context.Context, org, slug string) (bool, error) {
```

Comment:
@OrlovM I can't find where this method is used

Reply:
This method is supposed to be used in other part of #58.
Added information to pull request description.